torch/testing/_internal/optests/autograd_registration.py

```diff
@@ -83,15 +83,18 @@ def autograd_registration_check(op, args, kwargs):
 
     # Determine which AutogradBACKEND key to check
     all_device_types = {arg.device.type for arg in all_tensors}
-    if not all_device_types.issubset(["cpu", "cuda"]):
+    # Added mps device.
```

Comment:
Please delete this comment, it only makes sense in the context of this PR

Reply:
Done.